docs/concepts.md

```diff
@@ -81,24 +117,33 @@ or acquire new bundles in real time.
 
 ## Cluster
 
-An Arlon cluster, also known as workload cluster, is a Kubernetes cluster
+An Arlon cluster, also called a 'workload cluster', is a Kubernetes cluster
 that Arlon creates and manages via a git directory structure stored in
 the workspace repository.
 
-(Under construction)
-
-## Cluster spec
+## Cluster Specification
 
-A cluster spec contains desired settings when creating a new cluster.
+The Cluster Specification, also called the 'Base Cluster' contains the desired settings when creating a new cluster.
```

Comment:
Cluster Spec and Base Cluster are not identical, even if they have similarities. Why merge the concepts here? And why do all of this if this is going to be replaced by Cluster Templates?

Reply:
okay I fixed it to keep both in the concepts document. please take another look